Controller/ImagineController.php

```diff
@@ -155,4 +178,14 @@ private function createRedirectResponse(\Closure $url, string $path, string $fil
             ]), 0, $exception);
         }
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return bool
+     */
+    private function isSupportWebp(Request $request): bool
```

Comment:
super minor thing, could we rename this method to be more "readable" in an english sentence.

f.e. `isWebpSupportEnabled()` ?

in other places in your code you use the variable `$isWebpSupported`, I guess this would then also be `$isWebPSupportEnabled`

Reply:
It seems to me that the word "enabled" is not very suitable here. It's about browser support, not flagging. I think something like `isBrowserIncludedWebpSupport()` would be more appropriate.